egs/discophone/v1/run_gp.sh

```diff
@@ -0,0 +1,384 @@
+#!/bin/bash
```

Comment:
I don't think we need this script - instead, copy `conf/lang.conf` to `conf/lang_gp.conf` and run `./run.sh --langs-config conf/lang_gp.conf`

Reply:
> I don't think we need this script - instead, copy `conf/lang.conf` to `conf/lang_gp.conf` and run `./run.sh --langs-config conf/lang_gp.conf`

You're right. I will clean it up and discard run_gp.sh. It was used only for my convenience during my experiments.